src/components/SelectOption/SelectOption.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import usersFromServer from '../../api/users';
```

Comment:
[CHECKLIST ITEM 5] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components for data that could be passed down as props.